save-frontend/src/main/kotlin/com/saveourtool/save/frontend/App.kt

```diff
@@ -31,109 +32,78 @@ import kotlinx.coroutines.await
 import kotlinx.coroutines.launch
 import kotlinx.serialization.json.Json
 
-/**
- * Top-level state of the whole App
- */
-external interface AppState : State {
-    /**
-     * Currently logged-in user or null
-     */
-    var userInfo: UserInfo?
-}
-
-/**
- * Main component for the whole App
- */
-@JsExport
-@OptIn(ExperimentalJsExport::class)
-class App : ComponentWithScope<PropsWithChildren, AppState>() {
-    init {
-        state.userInfo = null
-    }
+@Suppress(
+    "EMPTY_BLOCK_STRUCTURE_ERROR",
+    "TOO_LONG_FUNCTION",
+    "LongMethod",
+    "ComplexMethod",
+)
+val app = VFC {
```

Comment:
Just in case: it is now `FC<PropsWithChildren>`

Reply:
yep, merged